release_mac.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 set -ex
 
-version=$(grep -m1 -Eo "[0-9]+\.[0-9]+\.[0-9]+" cmd/web3/version.go)
+#need to autoincrement because circle ci use the same git revision as on the previous step
+majorVersion=$(grep -m1 -Eo "[0-9]+\.[0-9]+" cmd/web3/version.go)
+minorVersion=$(grep -m1 -Eo ".[0-9]+\"" cmd/web3/version.go|grep -m1 -Eo "[0-9]+")
+version="$majorVersion.$(($minorVersion + 1))"
```

Comment:
Isn't it `major.minor.patch`? What happens if the commit message bumps the minor version, not the patch? Won't this diverge?

Reply:
nope, it's just replicate behaviour of this code - https://github.com/gochain-io/web3/blob/master/release.sh#L20 
The problem that on the previous step circle bumps the version and commit the code but on this step it doesn't checkout the new code but use the same revision as on the previous step, so had to perform the same bump, the reason of why I didn't use docker because I had to install it, so easier to implement with 3 lines of bash code (I think could replace the code in the previous step as well)

Reply:
The preceding comment suggests that it does not only bump the patch version: https://github.com/gochain-io/web3/blob/master/release.sh

Reply:
It's supposed to recognize "bump major" or "bump minor" from the commit message

Reply:
updated